src/Api/Pagination/ApiPaginator.php

```diff
@@ -33,7 +33,7 @@ class ApiPaginator
      */
     protected $total;
 
-    public function __construct(ApiQuery $query, $perPage = 20, $page = 1)
+    public function __construct(ApiQuery $query, $perPage = 100, $page = 1)
```

Comment:
we shouldn't be changing this library. You're changing the default pagination setting for all users of this library, we should instead handle this in the specific WHMCS call that needs this pagination setting.